### PR TITLE
BETA: Register scythemedia.is-a.dev

### PR DESCRIPTION
**Pull request by InariFoxed**
Added `scythemedia.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/media.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ScytheMediaTheDev",
+        "email": "tswartz22@aol.com"
+    },
+    "record": {
+        "MX": ["10"]
```
Comment (Stef-00012) [inline domains/media.json:7]: Invalid MX record

File: domains/scythe.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ScytheMediaTheDev",
+        "email": "tswartz22@aol.com"
+    },
+    "record": {
+        "URL": "https://www.scythe.is-a.dev"
```
Comment (Stef-00012) [inline domains/scythe.json:7]: That domain doesn't exist in the is-a.dev repo

File: domains/scythemail.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ScytheMediaTheDev",
+        "email": "tswartz22@aol.com"
+    },
+    "record": {
+        "MX": ["scythemail.is-a.dev"]
```
Comment (Stef-00012) [inline domains/scythemail.json:7]: You can't set the domain itself as MX record

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Comment by Stef-00012** [comment]
> how do i delete other domains @wdhdev @phenax

i saw that only after the review, just delete the file from your fork

**Comment by ScytheMediaTheDev** [comment]
you never actually accepted my pr for scythemedia.is-a.dev @Stef-00012 @wdhdev @phenax

**Comment by Stef-00012** [comment]
> you never actually accepted my pr for scythemedia.is-a.dev @Stef-00012 @wdhdev @phenax

the pr can't be merged as it contains other invalid domains, if you want only the `scythemedia.is-a.dev` domain to be merged, you need to first fix or remove the other invalid ones

**Comment by ScytheMediaTheDev** [comment]
ok

**Comment by ScytheMediaTheDev** [comment]
I know but i copied the code from it so its not lost @Stef-00012 , do i just paste the code into a file called (scythemedia.json)

**Comment by ScytheMediaTheDev** [comment]
is there a way to undo that commit of deleting the scythemedia.is-a.dev file?

**Comment by Stef-00012** [comment]
delete media.json and create scythemedia.json with the correct json & records

**Comment by ScytheMediaTheDev** [comment]
@Stef-00012 is that good now?
I made the json file and uploaded it
also deleted media.json

**Comment by ScytheMediaTheDev** [comment]
@wdhdev @phenax @Stef-00012 @is-a-dev/maintainers

**Comment by ScytheMediaTheDev** [comment]
Yo this system is messed up, @wdhdev @Stef-00012 @phenax

**Comment by wdhdev** [comment]
@ScytheMediaTheDev Please stop mentioning maintainers, we will get to your PR when we can.

**Comment by ScytheMediaTheDev** [comment]
ok

**Comment by ScytheMediaTheDev** [comment]
srry my bad, just waited at least a week or two to even get a reply

**Review by wdhdev — APPROVED** [review]

**Comment by ScytheMediaTheDev** [comment]
Thanks!
    On Monday, February 26, 2024 at 12:45:11 AM EST, reviewmate[bot] ***@***.***> wrote:  
 
 


Pull Request Merged

 

What's next?

Just follow the steps below and you can start using your domain!

GitHub Pages
   
   - Go to your GitHub repository that you use for your is-a.dev domain.
   - Click on the Settings tab.
   - Click on the Pages link in the sidebar.
   - In the custom domain option, enter the domain you registered.
   - Check the Enforce HTTPS checkbox.
   - Give your domain some time (~24 hours) for the changes to propagate.
   - Enjoy your domain!

Server Owners

If your domain points to a server you own, add your-domain.is-a.dev to your domain configuration. For HTTPS, you will have to configure an SSL certificate to allow the new subdomain to work (here's why).

Need help with your domain? If you have troubles setting up your domain, please create an issue and we will try and help you as soon as possible!

Made a mistake in the records? Don't worry, you can create a new pull request with the corrections.

**Pull request merged**